sdk/storage/azure-storage-blob-changefeed/samples/change_feed_samples.py

```diff
@@ -87,16 +89,76 @@ def list_events_using_continuation_token(self):
         token = change_feed.continuation_token
 
         # restart using the continuation token
-        change_feed2 = cf_client.list_changes(results_per_page=3).by_page(continuation_token=token)
+        change_feed2 = cf_client.list_changes(results_per_page=56).by_page(continuation_token=token)
         change_feed_page2 = next(change_feed2)
         for event in change_feed_page2:
             print(event)
 
+    def list_events_in_live_mode(self):
+        # Instantiate a ChangeFeedClient
+        cf_client = ChangeFeedClient("https://{}.blob.core.windows.net".format(self.ACCOUNT_NAME),
+                                     credential=self.ACCOUNT_KEY)
+        # to get continuation token
+        start_time = datetime(2020, 8, 19, 10)
+        change_feed = cf_client.list_changes(start_time=start_time).by_page()
+
+        for page in change_feed:
+            for event in page:
+                print(event)
+        token = change_feed.continuation_token
+
+        sleep(120)
+        print("continue printing events")
+        # restart using the continuation token
+        change_feed2 = cf_client.list_changes(results_per_page=56).by_page(continuation_token=token)
+        change_feed_page2 = next(change_feed2)
+        for event in change_feed_page2:
+            print(event)
+
+        sleep(120)
+        print("continue printing events")
+
+        token2 = change_feed2.continuation_token
+        change_feed3 = cf_client.list_changes(results_per_page=56).by_page(continuation_token=token2)
+        change_feed_page3 = next(change_feed3)
+        for event in change_feed_page3:
+            print(event)
```

Comment:
Could we just add top level infinite loop ?

Reply:
Ultimately this should never end right? This sample should demonstrate to customer how to actively listen to the tail of CF.
I assume concern here is that python runs samples in automation right ? (if that the case we may cap top level loop iterations with some artificial counter OR find a way to skip this sample in live run).

Reply:
add an infinite loop, just the dicusssion about mutual exclusive is still open